MQTTClient/MQTTClient/MQTTWebsocketTransport/MQTTWebsocketTransport.h

```diff
@@ -20,6 +20,11 @@
 */
 @property (strong, nonatomic) NSString *host;
 
+/** url an NSString containing the websocketurl
+ * defaults to @""
+ */
+@property (strong, nonatomic) NSString *url;
```

Comment:
Why not `NSURL` and nil by default?

Reply:
Made the changes. Let me know if they look fine.